capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,17 @@
+import java.util.List;
+
+interface CollegeScoreCardInterface {
+  /*General Use Functions*/
+  public boolean validateCollegeName(String collegeName);
```

Comment:
You should be able to omit public on all interface methods. They [default to public visibility](http://docs.oracle.com/javase/specs/jls/se7/html/jls-6.html#jls-6.6.1).

Reply:
done!